From 6ec2c60668edd79d78552c0345e40bab52e3aa33 Mon Sep 17 00:00:00 2001
From: Daniel Wagner <dwagner@suse.de>
Date: Wed, 14 Feb 2024 14:19:07 +0100
Subject: [PATCH] build: disable sysfs test

The test is not running stable in OSC environment. This is a new test
which is running stable in the upstream CI builds. So this is something
downstream and needs some more digging.

Signed-off-by: Daniel Wagner <dwagner@suse.de>
---
 test/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/meson.build b/test/meson.build
index 93e69991d6d7..a76f57973b6f 100644
--- a/test/meson.build
+++ b/test/meson.build
@@ -99,5 +99,5 @@ subdir('ioctl')
 subdir('nbft')
 
 if json_c_dep.found()
-  subdir('sysfs')
+#  subdir('sysfs')
 endif
-- 
2.43.0

